samples/js/js-eventsourced-shopping-cart/proto/shoppingcart_api.proto

```diff
@@ -56,22 +60,21 @@ service ShoppingCartService {
 
   rpc AddItem(AddLineItem) returns (google.protobuf.Empty) {
     option (google.api.http) = {
-      post: "/cart/{cart_id}/items/add"
-      body: "*"
+        post: "/cart/{user_id}/items/add"
```

Comment:
To change to `{user_id}` the field in `AddLineItem` would also need to be changed. This should have an error currently, as it won't find the field.

Reply:
That was a copy/paste error 😅